src/main/java/pl/edu/mimuw/Main.java

```diff
@@ -39,24 +41,270 @@ private static Scene createSampleScene(int size) {
   }
 
   private static VBox createSampleButtons(int size, Consumer<String> onButtonClicked) {
-    final HBox[] rows = new HBox[size];
+    final HBox[] rows = new HBox[size + 1];
+    int i = 0;
+    StringBuilder sb = new StringBuilder();
+    ArrayList<String> actions = new ArrayList<>();
+    AtomicReference<String> action = new AtomicReference<>(new String());
```

Comment:
to wyglada bardzo podejrzanie i nie jest chyba nigdzie używane, więc nie powinno tego być 😉